include/aarch64/gpio.h

```diff
@@ -0,0 +1,13 @@
+#ifndef _GPIO_H_
+#define _GPIO_H_
+
+struct resource_t;
+
+/* Select GPIO alt function. For more information look at
+ * https://cs140e.sergio.bz/docs/BCM2837-ARM-Peripherals.pdf and
+ * https://pinout.xyz */
+void gpio_function_select(resource_t *r, unsigned pin, unsigned func);
```

Comment:
Should type of `func` be enum with `BCM2835_GPIO_*` ?

Reply:
Done.